README.md

```diff
@@ -35,13 +37,28 @@ curl -X POST --header 'Content-Type: application/json' http://127.0.0.1:8081/ser
           "version": "1"
         }
     ]
-}' 
+} 
+```
+
+Searching tags:
 ```
+GET http://127.0.0.1:8081/service/rest/v1/tags?attribute=status%3Asuccessful&associatedComponent=repo1%3Agr1%3Acomp1%20%3E%3D%201
```

Comment:
please write url decoded text here for readablity.
postman and browsers are smart enough to encode them during request.

Reply:
Done